### PR TITLE
Behat for Drupal 8: Adding the blog post.

### PR DESCRIPTION
**Pull request by RoySegall**
@amitaibu Can you review please?

### CODE REVIEW

File: app/_posts/2015-04-14-drupal8-behat.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+  }
+}
+```
+
+But how i can add it to Drupal 8 autoloader? I looked for several blog posts but
+ non of them gave me the answer. The solution is very simple. Inside the module 
+ i file i included the autoloader generated by composer:
+
+```php
+<?php
```
Comment (amitaibu) [inline app/_posts/2015-04-14-drupal8-behat.md]: you don't need the `<?php` tag

File: app/_posts/2015-04-14-drupal8-behat.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+  "require": {
+    "Behat/gherkin": "~4.3"
+  },
+  "config": {
+    "vendor-dir": "src/Behat"
+  }
+}
+```
+
+But how i can add it to Drupal 8 autoloader? I looked for several blog posts but
```
Comment (amitaibu) [inline app/_posts/2015-04-14-drupal8-behat.md]: there is no word ``i`` in English :)

File: app/_posts/2015-04-14-drupal8-behat.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+published: true
+---
+
+{% include setup %}
+The first time i heard about Behat was at DrupalCon Munich 2012. Since then the usage of Behat has grown exponential and the tools it can be integrate with has grown as well - we can test the markup of a page. Pretty amazing, no? Well, It’s time to take it a little bit higher but in another direction. But first, let’s see how it’s actually works.
+
+Behat built from several symfony components:
+
+  - Cucumber steps - AKA feature files(*.feature). Contains the scenarios.
+  - Scenario - Collection of steps written in plain english, or any other
```
Comment (amitaibu) [inline app/_posts/2015-04-14-drupal8-behat.md]: this looks in italic for some reason, maybe markdown error somewhere

File: app/_posts/2015-04-14-drupal8-behat.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+
+Pretty simple. Each feature file could contain a bunch of scenario and we might 
+want to run tests with tags. This is how:
+
+```php
+<?php
+	$this->setTag('@login-success');
+```
+
+This will make sure only scenarios with the @login-success tag will run. We need
```
Comment (amitaibu) [inline app/_posts/2015-04-14-drupal8-behat.md]: ``@login-success`` (surround with '`')

File: app/_posts/2015-04-14-drupal8-behat.md [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+lot more simple:
+  
+```php
+<?php
+  public function testLoginFailed() {
+    $this->setTag('@login-failed')->executeScenario('login', 'Behat');
+  }
+```
+
+The module available on [github](https://github.com/roysegall/Behat) and ready
```
Comment (amitaibu) [inline app/_posts/2015-04-14-drupal8-behat.md]: move link to somewhere in the top

**Comment by amitaibu** [comment]
Please add a screenshot that will appear somewhere in the first few paraghraphs

File: app/_posts/2015-04-14-drupal8-behat.md (hunk 3 above)
Comment (RoySegall) [inline app/_posts/2015-04-14-drupal8-behat.md]: Yeah, probably:
selection_392

**Comment by RoySegall** [comment]
@amitaibu Can you review again?

**Comment by amitaibu** [comment]
Can you please open it directly under the Gizra account - easier to edit

**Comment by amitaibu** [comment]
actually, never mind, I'll do it

**Comment by RoySegall** [comment]
I don't think have permission for pushing to Gizra/Gizra.

**Comment by amitaibu** [comment]
closed in favor of #151
